skore/src/skore/project/reports.py

```diff
@@ -0,0 +1,21 @@
+"""Module to manage the persisted reports."""
+
+from skore.externals._pandas_accessors import DirNamesMixin
+from skore.project import Project
+from skore.project.metadata import Metadata
+from skore.sklearn import EstimatorReport
```

Comment:
To avoid circular imports with my other comments:

```suggestion
from __future__ import annotations

from typing import TYPE_CHECKING

from skore.externals._pandas_accessors import DirNamesMixin

if TYPE_CHECKING:
    from skore.project import Project
    from skore.project.metadata import Metadata
    from skore.sklearn import EstimatorReport
```